scheduling/taskmetadata.go

```diff
@@ -172,23 +180,83 @@ func (m *TaskMetadataManager) CreateTaskRun(name string, taskID TaskID, trigger
 	if err != nil {
 		return TaskRunMetadata{}, err
 	}
-	err = m.storage.Create(taskRunKey.String(), string(serializedRuns))
+	err = m.Storage.Create(taskRunKey.String(), string(serializedRuns))
 	if err != nil {
 		return TaskRunMetadata{}, err
 	}
 
 	taskRunMetaKey := TaskRunMetadataKey{taskID: taskID, runID: metadata.ID, date: startTime}
-	err = m.storage.Create(taskRunMetaKey.String(), string(serializedMetadata))
+	err = m.Storage.Create(taskRunMetaKey.String(), string(serializedMetadata))
 	if err != nil {
 		return TaskRunMetadata{}, err
 	}
 
 	return metadata, nil
 }
 
+func (m *TaskMetadataManager) GetLatestRun(taskID TaskID) (TaskRunMetadata, error) {
```

Comment:
I made this on both sides of the grpc call. The one on the client reduces the # of endpoints, but this is probably much more effecient

Reply:
Will probably add another endpoint for this so we're not sending a ton of runs in a message

Reply:
Note that this isn't guaranteed to return the latest run since its not locking

Reply:
Think thats fine. Believe we only use this for visual things on the dashboard